add test for appsettings.json based config

monishkadas-ms opened the pull request:
Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

monishkadas-ms committed:
add test for appsettings.json based config

Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking
sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from
appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

monishkadas-ms committed:
add test for appsettings.json based config

Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking
sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from
appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

monishkadas-ms committed:
add test for appsettings.json based config

Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking
sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from
appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

monishkadas-ms committed:
add test for appsettings.json based config

Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking
sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from
appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

monishkadas-ms committed:
add test for appsettings.json based config

Updated the AzureDataExplorerSinkE2ETests.cs with a test for checking
sink configuration using appsettings.json.

Updated LoggerConfiguration to accept userToken as a parameter from
appsettings.json to allow userToken based Authentication Method.

Added additional packages and updated Serilog version

tanmaya-panda1 commented on src/Serilog.Sinks.AzureDataExplorer.Tests/AzureDataExplorerSinkE2ETests.cs:
```diff
@@ -196,6 +199,55 @@ public async Task Test_AzureDataExplorer_SerilogSink(string identifier, string r
         var actualExceptionIngested = GetExceptionIngested();
         Assert.Contains("A nested exception!", actualExceptionIngested);
         Assert.Contains("Percentages must be between 0 and 100", actualExceptionIngested);
+    
```
adding this AppSettingsConfiguration as a part of existing test cases is not a good idea. Please do create separate method to test this config.

monishkadas-ms replied:
Done!

tanmaya-panda1 committed:
Update batching policy of generated table

tanmaya-panda1 committed:
Removed dependency on Serilog.Sinks.PeriodicBatching Package

Batching is supported natively starting from Serilog v4.x. The Periodic
Batching package is no longer needed.

Upgraded the Sink Package Version.
